sqlglot/parser.py

```diff
@@ -306,6 +308,7 @@ class Parser(metaclass=_Parser):
         TokenType.ROW,
         TokenType.UNNEST,
         TokenType.VAR,
+        TokenType.HASH,
```

Comment:
Why is this needed here?

Reply:
Ah, forgot to remove that when reverting the tokens.